packages/next-swc/crates/next-core/src/app_source.rs

```diff
@@ -1018,7 +1018,7 @@ import {}, {{ chunks as {} }} from "COMPONENT_{}";
             project_path,
             intermediate_output_path,
             intermediate_output_path.join("chunks"),
-            server_root.join("_next/static/assets"),
+            server_root.join("_next/static/media"),
```

Comment:
We should store this as a `const` somewhere, or have a method to get it, instead of repeating it in three different places.